Assets/Scripts/Enemy/EnemyTest.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyTest : EnemyController
+{
+    //[SerializeField] protected GameObject playerObject;
+    protected int EnemyTestHealth = 20;
+
+    // Start is called before the first frame update
+    protected override void Start()
```

Comment:
All the functionality in this method can be done in parents

Reply:
I wanted enemies to have different maxHealth and different moveSpeed

Reply:
Let's create a Scriptable Object to hold enemy data later on